src/main/java/org/verdictdb/metastore/ScrambleMetaStore.java

```diff
@@ -115,6 +125,74 @@ public void addToStore(ScrambleMeta scrambleMeta) throws VerdictDBException {
     addToStore(scrambleMetaSet);
   }
 
+  public VerdictSingleResult showScrambles() throws VerdictDBException {
+    String tableAlias = "t";
+    SelectQuery query =
+        SelectQuery.create(
+            Arrays.<SelectItem>asList(
+                new BaseColumn(tableAlias, ORIGINAL_SCHEMA_COLUMN),
+                new BaseColumn(tableAlias, ORIGINAL_TABLE_COLUMN),
+                new BaseColumn(tableAlias, SCRAMBLE_SCHEMA_COLUMN),
+                new BaseColumn(tableAlias, SCRAMBLE_TABLE_COLUMN),
+                new BaseColumn(tableAlias, ADDED_AT_COLUMN),
+                new BaseColumn(tableAlias, DATA_COLUMN)),
+            new BaseTable(storeSchema, METASTORE_TABLE_NAME, tableAlias));
+    query.addOrderby(new OrderbyAttribute(ADDED_AT_COLUMN, "desc"));
+    String sql = QueryToSql.convert(conn.getSyntax(), query);
+    DbmsQueryResult result = conn.execute(sql);
+
+    return new VerdictSingleResultFromDbmsQueryResult(result);
+  }
+
+  public void dropAllScrambleTable(BaseTable originalTable) throws VerdictDBException {
+    String originalTableSchema = originalTable.getSchemaName();
+    String originalTableName = originalTable.getTableName();
+    ScrambleMetaSet metaSet = this.retrieve();
+
+    for (Iterator<ScrambleMeta> it = metaSet.iterator(); it.hasNext(); ) {
+      ScrambleMeta meta = it.next();
+      if (meta.getOriginalTableName().equals(originalTableName)
+          && meta.getOriginalSchemaName().equals(originalTableSchema)) {
+        String scrambleTableSchema = meta.getSchemaName();
+        String scrambleTableName = meta.getTableName();
+        this.dropScrambleTable(
+            new BaseTable(originalTableSchema, originalTableName),
+            new BaseTable(scrambleTableSchema, scrambleTableName));
+      }
+    }
+  }
+
+  public void dropScrambleTable(BaseTable originalTable, BaseTable scrambleTable)
+      throws VerdictDBException {
+    String originalTableSchema = originalTable.getSchemaName();
+    String originalTableName = originalTable.getTableName();
+    String scrambleTableSchema = scrambleTable.getSchemaName();
+    String scrambleTableName = scrambleTable.getTableName();
+
+    // drop the actual scrambled table first.
+    DropTableQuery dropQuery = new DropTableQuery(scrambleTableSchema, scrambleTableName);
+    dropQuery.setIfExists(true);
+    String sql = QueryToSql.convert(conn.getSyntax(), dropQuery);
+    conn.execute(sql);
+
+    // update metadata with a new row where 'data' and 'method' marked as 'DELETED'
+    InsertValuesQuery insertQuery = new InsertValuesQuery();
+    insertQuery.setSchemaName(getStoreSchema());
+    insertQuery.setTableName(getMetaStoreTableName());
+    String timeStamp = new SimpleDateFormat(TIMESTAMP_FORMAT).format(new Date());
+    insertQuery.setValues(
+        Arrays.<Object>asList(
+            originalTableSchema,
+            originalTableName,
+            scrambleTableSchema,
+            scrambleTableName,
+            DELETED,
```

Comment:
The scramble table seems to have 6 columns, but this insert statement has 7 values.

Reply:
it actually has 7 columns like the following:
```
+-----------------+-----------+---------+
| name            | type      | comment |
+-----------------+-----------+---------+
| original_schema | string    |         |
| original_table  | string    |         |
| scramble_schema | string    |         |
| scramble_table  | string    |         |
| scramble_method | string    |         |
| added_at        | timestamp |         |
| data            | string    |         |
+-----------------+-----------+---------+
```